docs/trace/building-your-own-exporter/MyExporterExtensions.cs

```diff
@@ -14,12 +14,28 @@
 // limitations under the License.
 // </copyright>
 
+using System;
 using OpenTelemetry.Trace;
 
-internal static class TracerProviderBuilderExtensions
+internal static class MyExporterExtensions
 {
+    public static TracerProvider AddMyExporter(this TracerProvider provider)
```

Comment:
Zipkin & Jaeger use "Use" not "Add" for their extensions. Not sure if there is a reason for that. We should probably have a convention something like: "Add" for sources/instrumentation, "Use" for processors/exporters/samplers, & "Set" for resource(s)?

Reply:
I _guess_ historically they picked `Use` since if multiple calls are made, the last one will win (and the previous ones would have no effect besides leaking the Dispose call, which is surprising).
The reason here I picked `Add` is the embrace the same pattern of processor, multiple exporters can be added.

Net net, my proposal:
* "Set" for resource and sampler - and we probably should have a better handling if multiple `Set` calls happened (e.g. how to handle Dispose, should we throw, etc.).
* "Add" for processor, exporter and instrumentation.